Update SonarQube step config doc: View scan results in SonarQube portal (#7402)
harness/developer-hub: docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference.md

- `JVM_HTTPS_PROXY_PORT : 3745`
- `JVM_NO_PROXY : sonar.myorg.local`

## View scan results in SonarQube portal
The scan results for your targets can be accessed in the SonarQube portal. These results are organized under projects titled with your target/repository name. Further, you can view scan results specific to branches, PRs, and tags based on the scans performed in STO.

### Locate STO scan results in SonarQube portal

1. **_Branch_ scan results:** If you performed a scan for a branch, you can find the results in SonarQube with the same branch name.
2. **Pull Request scan results:** If you performed a scan for a PR, the results will be located under the name `refs/pull/<PR_NUMBER>`, where `<PR_NUMBER>` is the specific number of the pull request.
3. **Tag scan results:** If you performed a scan for a tag, the results will be available with the same tag name.

### Set the name for your scans

When using the scan configuration "Branch Scan," you can set the naming of the scan as it will appear in the SonarQube portal. This can be configured as follows:

> 1. Set the “Analysis Type” to "manual" under the “Scan Tool” section. This brings up a new field called "Branch Name".
> 2. In the "Branch Name" field, you can define the desired name for the scan result as it should appear in the SonarQube portal.
> 

Reviewer: ```suggestion
1. Under **Scan Tool**, set the **Analysis Type** to **Manual**. 
2. Set the **Branch Name** to the name you want to use for your scan results in the SonarQube portal.
```

I think you can tighten this up. Often you don't need to say "the ___ section."  Here's a good discussion from the Microsoft style guide: 
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

Also, don't use the word "desire" -- use "want" instead.

https://developers.google.com/style/word-list#desire

<DocImage path={require('./static/sonarqube-orchestration-branch-scan.png')} width="30%" height="30%" title="Click to view full size image" />
<!-- ![alt_text](./static/sonarqube-orchestration-branch-scan.png "image_tooltip") -->

### Best practices

> You can utilize [Harness expressions](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/) to automatically populate values from triggers. For example, you can define the "Branch Name" as `<+trigger.sourceBranch>/<+trigger.prNumber>`. This will generate a naming convention in the format SOURCE_BRANCH_NAME/PR_NUMBER, which is particularly useful when triggering scans on a specific PR. 

Reviewer: ```suggestion
You can use [Harness expressions](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/) to automatically populate values from triggers. For example, you can define the "Branch Name" as `<+trigger.sourceBranch>/<+trigger.prNumber>`. This will generate a naming convention in the format SOURCE_BRANCH_NAME/PR_NUMBER, which is particularly useful when triggering scans on a specific PR. 
```

I think you mean "use" here, not "utilize"....
https://developers.google.com/style/word-list#utilize
